src/block_cache/block_mgr.c

```diff
@@ -182,6 +182,69 @@ __bm_checkpoint_unload(WT_BM *bm, WT_SESSION_IMPL *session)
     return (ret);
 }
 
+/*
+ * __bm_close_block_remove --
+ *     Remove a single block.
```

Comment:
Is there an alternate name for a `WT_BLOCK`? Might people be confused because we also talk about reading and writing individual blocks?  

This is probably baked into WT code and terminology. So feel free to ignore if I'm howling at the moon here.

Reply:
+1 this was confusing for me getting started with this code.

Reply:
Fixed in ccc845096.

At the moment, `WT_BM` is a block manager a.k.a "file", and `WT_BLOCK` is a block interface handle, a.k.a. "file", so this needs cleanup. They should probably merge because there's no reason to believe different objects will have the same methods for access.

Reply:
Good to pursue in a different ticket, I think.

Reply:
+1 Same here. I've been confused about this since I saw this terminology.

Reply:
FTR, I'm likely to simply merge the `WT_BM` structure into the `WT_BLOCK` structure. I see both the block cache and block layers as "serving blocks", so I think `WT_BLOCK` makes sense.